legend-engine-xts-relationalStore/legend-engine-xt-relationalStore-generation/legend-engine-xt-relationalStore-pure/legend-engine-xt-relationalStore-sqlDialectTranslation-pure/src/main/resources/core_external_store_relational_sql_dialect_translation/sqlDialect.pure

```diff
@@ -27,6 +28,7 @@ Class meta::external::store::relational::sqlDialectTranslation::SqlDialect
   identifierProcessor: IdentifierProcessor[1];
   expressionPrecedenceComparator: ExpressionPrecedenceComparator[1];
   keywords: String[*];
+  functionProcessorDispatch: Function<{SqlDialect[1], SqlDialectTranslationState[1], SqlDialectTranslationConfig[1] -> Map<Class<SqlFunction>, FunctionProcessor>[1]}>[0..1];

```

Comment:
Type should be a Map<Class<SqlFunction>, FunctionProcessor>[1] instead of function returning it. At the time of instantiation, map should be created, and same map should be used for all function translations with the dialect. Instantiation of the map should not be done multiple times.